### PR TITLE
POC Frameworks

### PR DESCRIPTION
**Pull request by jacek-oet**
Streamlit vs Plotly/dash

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+- pip: 24.1.2
+
+## Run Project
+   1. Create folder "venv"
+      ```shell
+      python -m venv venv
+      ```  
+   2. Install 
+      ```shell
+      pip install streamlit
```
Comment (siddharth-krishna) [inline README.md:19]: You need to activate the venv before installing pip dependencies, btw

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+      .\venv\Scripts\activate
+      ```
+   4. Run project
+      - streamlit
+         ```shell
+         streamlit run pocs\streamlit\app.py
+         ```
+      - plotly/dash
+         ```shell
+         py pocs\dash\app.py
```
Comment (siddharth-krishna) [inline README.md:35]: py = python?

**Review by siddharth-krishna — APPROVED** [review]
Looks good, thanks. Please also check the comments on #2 and implement them in this one before merging.

File: README.md (hunk 1 above)
Comment (jacek-oet) [reply README.md]: I updated.

File: README.md (hunk 2 above)
Comment (jacek-oet) [reply README.md]: I updated.

**Pull request merged**